shared/src/business/useCases/trialSessions/getCalendaredCasesForTrialSessionInteractor.ts

```diff
@@ -31,14 +31,19 @@ export const getCalendaredCasesForTrialSessionInteractor = async (
 
   // instead of sending EVERY docket entry over, the front end only cares about the PMT documents not stricken
   // to figure out the filingPartiesCode
-  const casesWithMinimalRequiredDocketEntries = cases.map(aCase => ({
-    ...aCase,
-    docketEntries: aCase.docketEntries.filter(
-      docketEntry => docketEntry.eventCode === 'PMT' && !docketEntry.isStricken,
-    ),
-  }));
+  const casesWithMinimalRequiredInformation = cases
+    .map(aCase => ({
+      ...aCase,
+      docketEntries: aCase.docketEntries.filter(
```

Comment:
I did some research into why we need to know if each case has a Pretrial Memorandum filed on it (that's also not stricken). There was a [story](https://github.com/flexion/ef-cms/issues/9512) implemented that added PMT as a column to the trial session working copy page which indicates if a PMT has been filed on each case and by whom. I think we could better document the business use case here by just returning `servedPartiesCode` for that docket entry if it exists.

Reply:
Good point!

Reply:
This refactoring has been rolled into the CalendaredCase entity, for now